dev/test-zeekoe.py

```diff
@@ -121,39 +121,39 @@ def run_command(cmd, verbose):
 
 def start_merchant_server(merch_config, verbose):
     info("Starting the merchant server...")
-    cmd = ["./target/debug/zkchannel", "merchant", "--config", merch_config, "run"]
+    cmd = ["../target/debug/zkchannel", "merchant", "--config", merch_config, "run"]
```

Comment:
Could you make this a constant at the top of the file?

Reply:
done